grpcbp/server_middlewares.go

```diff
@@ -154,7 +154,7 @@ func StartSpanFromGRPCContext(ctx context.Context, name string) (context.Context
 //
```

Comment:
since you removed `service` suffix from `spectest`, we need to add the tests for `gprc_service` label here manually. from my quick glance of the code I don't believe we are producing the expected `grpc_service` label values.

Reply:
you are correct. the spec test currently only checks the thrift metrics. i need to add support for grpc and http. i think i will do that in a separate PR if thats ok.

Reply:
that means this PR can produce totally wrong labels and that will only get fixed by the next PR that adds the tests, which doesn't sound ideal to me.

Reply:
ok i will add the spec test changes here to include grpc